marko.json

```diff
@@ -182,5 +182,17 @@
             "@selected": "boolean"
         }
     },
-    "<ebay-select-option>": {}
+    "<ebay-select-option>": {},
+    "<ebay-textbox": {
+        "renderer": "./src/components/ebay-textbox/index.js",
+        "@*": "expression",
+        "@html-attributes": "expression",
+        "@value": "string",
+        "@class": "string",
+        "@placeholder": "string",
```

Comment:
Similar for this one, it seems like it would just be a pass-through. You should still show these attributes in examples since they are required for most usage, but we don't need to handle it through Marko attributes.